CHANGELOG/CHANGELOG-2.3.md

```diff
@@ -6,6 +6,16 @@ When cutting a new release, update the `unreleased` heading to the tag being gen
 
 ## Unreleased
 
+## v2.3.1 - 2024-11-08
+
+### New Features
+
+* [#130](https://datastax.jira.com/browse/ZDM-616): Security vulnerabilities in ZDM version 2.3.0
```

Comment:
this JIRA project is not public, I don't think we're supposed to link JIRA issues here, JIRA is only meant for internal tracking and is synced automatically, everything should be done via GH issues and PRs

Reply:
Replaced with GH issue number.